### DISCUSSION TITLE
Should LosslessCut have a repo wiki?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [mifi/lossless-cut#1542] [Ideas]
I haven't used LosslessCut that much and am not aware of the many ways that users use it (or if there are even that many different uses), but I think it would be good if there was a wiki that goes over the UI and all the different buttons and menu items. It's mentioned [here](https://github.com/mifi/lossless-cut/issues/1536#issuecomment-1496926335) that its possible to turn off project file saving but I couldn't figure out how from looking at the app so I replied back.

**Comment by mifi** [comment]
Not sure what are the benefitd of a wiki over just markdown files, but it’s something to consider. If someone wants to document more features that would be most welcome, as I have limited time.
